<?xml version="1.0"?>
<launch>
  <node name="draw_OBBs" pkg="ros_tools" type="draw_OBB_node" cwd="node">
  <param name="load_background" value="false"/>
  <param name="background_path" value="/home/hoang/datasets_online/office-iros/background.ply"/>
  <param name="data_dir" value="/home/hoang/Desktop/"/>
  <param name="color_segment" value="false"/>
  <rosparam param="objects_name"> [011_banana, 044_flat_screwdriver]</rosparam> 
  </node>
</launch>